first_5_steps/1_install_tt_buda.md

```diff
@@ -31,7 +31,9 @@ Once you have identified the release version you would like to install, you can
    6. [Tenstorrent Software Package](#tenstorrent-software-package)
```

Comment:
I think we should remove the Tenstorrent Software Package section (see other comment below).

And I think we should have a header that says something like "PyBuda Installation" and have the next two sections be subsections.

Something like this -- open to making it nicer:
```markdown
6. [PyBuda Installation]
  i. [Python Environment Installation]
  ii. [Docker Container Installation]
```